BenchmarkVCFs/FindSamplesAndBenchmark.wdl

```diff
@@ -134,10 +139,10 @@ workflow FindSamplesAndBenchmark {
                      jexlVariantSelectors = jexlVariantSelectors,
                      variantSelectorLabels = variantSelectorLabels,
                      referenceVersion = "1",
-                     doIndelLengthStratification=false,
+                     doIndelLengthStratification=doIndelLengthStratification,
                      gatkTag="4.0.11.0",
-                     requireMatchingGenotypes=true,
-                     passingOnly=true,
+                     requireMatchingGenotypes=requireMatchingGenotypes,
```

Comment:
Should we also push the referenceVersion and GATK tags up?

Reply:
done